lib/dataContract/validateDataContractFactory.js

```diff
@@ -221,6 +222,25 @@ module.exports = function validateDataContractFactory(
                 ),
               );
             });
+
+          if (result.getErrors().length > 0) {
```

Comment:
It could be errors from different index so you need to return after adding ”property not found” error